documentation/user-interface/main-menu.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: docs
+title: Main Menu
+permalink: /documentation/user-interface/main-menu.html
+---
+# Main Menu
+
+The main menu contains the following items:
+
+* [File](#file)
+* [Edit](#edit)
+* [View](#view)
+* [Help](#help)
+
+## File
+
+The **File** menu provides the following commands:
+
+* **Open Test Directory** - Opens a [test directory](../guides/organize-tests.md#test-directory).
+* **Close Test Directory** - Closes a currently opened test directory.
+* **Recent** - Contains a submenu that lists recently opened test directories.
+* **Exit** - Closes TestCafe Studio.
+
+## Edit
+
+The **Edit** menu provides the following commands:
+
+* **Undo** - Undoes the last operation.
+* **Redo** - Reverts a cancelled operation.
+* **Cut** - Cuts selected text and copies it to the clipboard.
+* **Copy** - Copies selected text to the clipboard.
+* **Paste** - Pastes a text from the clipboard to a current cursor position in a text editor.
+* **Select All** - Selects all text in an editor.
+
+## View
+
+The **View** menu provides the following commands:
+
+* **Enter Full Screen** - Resizes the TestCafe Studio window to fit the screen.
+* **Show/Hide Explorer** - Displays or hides the [Explorer](explorer-panel.md) panel.
+* **Show/Hide Reports** - Displays or hides the [Reports](reports-panel.md) panel.
+* **Show Previous Tab** - Activates a tab to the left of a currently active tab.
+* **Show Next Tab** - Activates a tab to the right of a currently active tab.
+
+## Help
+
+The **Help** menu provides the following commands:
+
+* **Welcome** - Displays the **Welcome** page.
```

Comment:
Let's add link to Welcome page section here

Reply:
I deleted this link because we don't know when we merge the Welcome topic.